src/users/types.ts

```diff
@@ -11,6 +11,10 @@ export type UserDB = {
   id: string;
   roleNames: string[];
   active: boolean;
+
+  createdAt?: Date;
+  lastUsedAt?: Date;
```

Comment:
Will we drop the `At` here or leave it?

Reply:
Leave it 👌 

see https://weaviate-org.slack.com/archives/C064AL0Q6F2/p1746527397954569?thread_ts=1745570462.670529&cid=C064AL0Q6F2